### PR TITLE
SLING-11734 - Use the groupId in the parent artifactId

### PR DESCRIPTION
**Pull request by niekraaijmakers**
Use the groupId in the parent artifactId to attempt to more accurately pick the right artifactId pom.properties file.

https://issues.apache.org/jira/browse/SLING-11734

### CODE REVIEW

File: src/test/java/org/apache/sling/feature/scanner/impl/ContentPackageScannerTest.java [hunk 1]
```diff
@@ -56,6 +56,19 @@ public void testScan() throws URISyntaxException, IOException {
             assertNotNull(desc.getManifest());
         }
     }
+    
+    @Test
+    public void testMultipleMavenPropertyDirectoryPicking() throws URISyntaxException, IOException {
+        // this test case is to cover where
```
Comment (kwin) [inline src/test/java/org/apache/sling/feature/scanner/impl/ContentPackageScannerTest.java:62]: incomplete sentence

**Comment by niekraaijmakers** [comment]
@kwin can you re-review? Thanks!

**Review by kwin — COMMENTED** [review]
I would appreciate that instead of committing binaries for test cases we could maybe reuse something which is published to Maven Central, like ACS AEM Commons (or maybe some smaller example) and just download that prior to the test with https://maven.apache.org/plugins/maven-dependency-plugin/copy-mojo.html

**Comment by cziegeler** [comment]
I think this needs a more complicated fix, see https://issues.apache.org/jira/browse/SLING-11734

**Comment by kwin** [comment]
Superseded by #43

**Pull request closed without merging**